src/krbd.cc

```diff
@@ -232,6 +232,8 @@ static int wait_for_udev_add(struct udev_monitor *mon, const krbd_spec& spec,
                              string *pname)
 {
   struct udev_device *bus_dev = nullptr;
+  list<struct udev_device*> block_dev_list;
```

Comment:
This can be a vector, no need for a list.

Prefix it with std, `std::vector<struct udev_device *>`.